azurerm/internal/services/containers/kubernetes_addons.go

```diff
@@ -154,6 +157,20 @@ func schemaKubernetesAddOnProfiles() *schema.Schema {
 						},
 					},
 				},
+
+				"open_service_mesh": {
```

Comment:
Can we document this new block [both in the `addon_profile` block](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/kubernetes_cluster.html.markdown#L198-L216) and the specific field within it ([example](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/kubernetes_cluster.html.markdown#L268-L274))? Notably since this is in Preview we'll also want to add this below both of those:

```
~> **Note:** Open Service Mesh is available on an opt-in Preview basis - to use this the Preview Feature `AKS-OpenServiceMesh` will need to be registered, [see the AKS Documentation for more information](https://docs.microsoft.com/en-gb/azure/aks/servicemesh-osm-about)
```

Reply:
@tombuildsstuff Added, thanks